.github/workflows/mediasoup-node.yaml

```diff
@@ -12,6 +12,7 @@ jobs:
   ci:
     strategy:
       matrix:
+        fail-fast: true
```

Comment:
This should have been at the level higher here, but also it defaults to `true`, so I don't think it is really a necessary change: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
Which could explain why all those CI jobs are not even being runned XDDD. Thanks, fixing.

Reply:
OMG it's not being fixed... why this??

<img width="905" alt="CleanShot-2023-12-29-at-10 33 15@2x" src="https://github.com/versatica/mediasoup/assets/16191/0c73a56e-f1bd-4570-a56e-e41a3911c5c3">

Reply:
ok typo introduced:

<img width="1263" alt="CleanShot-2023-12-29-at-10 34 31@2x" src="https://github.com/versatica/mediasoup/assets/16191/3d3f619a-38c5-4bda-aec1-275080df2864">